source/about/mattermost-v10-changelog.md

```diff
@@ -4,6 +4,91 @@
+ - Added a browser window title to the Scheduled Posts tab. The title is ``Scheduled - <team name>``, same pattern as in the Drafts tab.
+
+#### Administration
+ - Added a new System Console page called **Embedding** which allows frame ancestor domains to be specified when embedding Mattermost in other web sites. Note, ``teams.microsoft.com`` is no longer added automatically to the frame ancestors list.
+ - The Channel Export plugin is removed from the transitional package list as it is now pre-packaged.
+ - Removed unnecessary log messages by checking the license before calling to retrieve groups.
+ - Made configuration location in the Support Packet human-readable.
+ - Added advanced audit and notifications logs to the Support Packet.
+ - Added log information to LDAP sync about ``include_removed_members`` option.
+ - Upgraded ``react-select`` from v3.0.3 to v5.9.0.
+
+### Bug Fixes
+ - Fixed an issue with the alignment of the draft list when scheduled posts was disabled.
+ - Fixed an issue where threads created by users were auto-followed on reply by the creator when they left the channel.
+ - Fixed an issue where muted channels in other teams would show their mentions in the title bar.
+ - Fixed an issue where messages from new channels in other teams wouldn't show up until a refresh.
+ - Fixed an issue with the scrolling behavior when navigating the Direct Message list using UP/DOWN arrow keys.
+ - Fixed a few minor bugs with websocket reconnection logic in the webapp.
+ - Fixed an issue where DND statuses did not expire at the expiry time displayed in the app.
+ - Fixed an issue where the group mentions permission was missing.
+ - Fixed an issue where a system bot reply to a command entered in a thread was also posted in the channel.
+ - Fixed an issue where the channel member menu might open in the wrong direction.
+ - Fixed an issue where the edit post textbox sized incorrectly with the Grammarly browser extension installed.
+ - Fixed an issue where onclick was missing in the channel header text, thus enabling hashtag, link, and mention clicks.
+ - Fixed an issue with jobs in a High Availability environment, where two job servers would take the same job.
+
+### config.json
+New setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
+ - Under ``MetricsSettings`` in ``config.json``:
+    - Added ``ClientSideUserIds`` where users can set the user IDs that they want to track for client side webapp metrics. Note that the total number of userIDs have been capped to 5 for performance reasons. Otherwise Prometheus gets overwhelmed with high label cardinality. It is recommended not to keep changing this list too often to track different sets of users, otherwise Prometheus might start to have poor performance. 
```

Comment:
```suggestion
    - Added ``ClientSideUserIds`` where users can set the user IDs that they want to track for client-side webapp metrics. The total number of userIDs have been capped to 5 for performance reasons, otherwise Prometheus gets overwhelmed with high label cardinality. We recommend modifying this list infrequently to ensure Prometheus performance. 
```